scripts/helpers.js

```diff
@@ -2,124 +2,119 @@
 
 'use strict';
 
-var pathFn = require('path');
-var _ = require('lodash');
-var cheerio = require('cheerio');
-var lunr = require('lunr');
+const pathFn = require('path');
+const _ = require('lodash');
```

Comment:
would love to see this replaced by native, but perhaps in a separate PR.

Reply:
`cheerio` here is use to add anchor for each headings.

Maybe we should leave it here. After we replace cheerio in `toc()` helper with native (https://github.com/hexojs/hexo/issues/3677), we might have more exprience dealing with this.

Reply:
I might open a separate PR, replacing lodash with native according to these:

- https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_each
- https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_assign
- https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_sortby-and-_orderby